openstudiocore/resources/analysis/pat_export/scripts/StandardReports/measure.rb

```diff
@@ -1,139 +1,165 @@
 require 'erb'
+require 'json'
+
+require "#{File.dirname(__FILE__)}/resources/os_lib_reporting"
+require "#{File.dirname(__FILE__)}/resources/os_lib_schedules"
+require "#{File.dirname(__FILE__)}/resources/os_lib_helper_methods"
 
 #start the measure
-class StandardReports < OpenStudio::Ruleset::ReportingUserScript
+class OpenStudioResults < OpenStudio::Ruleset::ReportingUserScript
```

Comment:
Here are the cases to consider:

1) existing OSM/OSP, this will reference the old standard report measure by name and.or uuid.  I thought we would want to keep the old one around for this reason.  When an old OSP is upgraded the code in SimpleProject would look for the new reporting measure and add it as well.
2) new OSM/OSP.  The new measure would be added only, the old measure would be around for people to drag into their workflows if they wanted to